recipes/marisa-trie/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "marisa-trie" %}
```

Comment:
Generally, I have been moving away from asking for prefixing for all sorts of things as it got kind of messy. Though in this case where both libraries have the same name I would kind of want to see it. That being said, I'm not too dogmatic about it.